projects/addon-editor/components/toolbar-new/toolbar-new.template.html

```diff
@@ -1,3 +1,4 @@
+<button class="t-prevent-parent-label-behavior"></button>
```

Comment:
Let's leave a comment or this might turn into wisdom of the old gods in a couple of years :D

Reply:
fixed